### DISCUSSION TITLE
Different number of IVs for treatments in simultaneous analysis

### DISCUSSION THREAD

**Discussion by hadigilan** [DoubleML/doubleml-for-r#152] [Q&A] [answered]
Hi all,

very briefly:
Suppose we have three treatments in a simultaneous analysis. But we want to input only two instrumental variables for the first and third treatments. How can we do that?

Thanks for your response,

**Comment by PhilippBach** [comment, marked as answer]
Hi @hadigilan ,

thanks for your interest in our package! 

Currently, we have only implemented the case that all instruments Z are used for all endogeneous treatments D in the multiple (endogenous) treatment case. In case you want to deviate from this, I'd suggest to implement each of the IV models separately, i.e., set up a data `DoubleMLData` backend and and estimate `DoubleMLPLIV` model separately for each of the treatment variables (say d1 and d3) separately. This should already give you valid IV-results, I think. 

In case you want to also use DoubleML's methods for simultaneous inference, things are getting a bit more complicated.  Either you'd have to manually merge the objects and/or modify the multiplier bootstrap for adjustment of p-values (`p_adjust()`) and joint confidence intervals (`confint()`); Or you'd try to go with bonferroni or p-value based approaches instead (like using `stats::p.adjust()` function for the resulting p-values) for valid simultaneous inference.

Hope that's helping you already a bit.

**Comment by hadigilan** [comment]
Thank you for the information!
It is certainly helpful.

Sincerely,
Hadi
